### PR TITLE
 Fix for issue #468 includes passing test

### PR DESCRIPTION
**Pull request by diberry**

### CODE REVIEW

File: test/issues/0468.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+
+var assert = require('assert');
+var yaml = require('../..');
+
+test('should not indent arrays an extra level when disabled', function () {
+  var output = yaml.dump([{"a":"a_value","b":"b_value"},{"a":"a2_value","expanded":true,"items":[{"a":"a_a_value","b":"a_b_value"}]}], { noArrayIndent: true });
```
Comment (puzrin) [inline test/issues/0468.js]: It's better to write such objects in nested form, starting from newline:

```js
var output = yaml.dump(
  [
    {
      a: "a_value",
....
  ],
```

Also probably worth shorten `a_value` => `a_val` to minimize useless noise.

File: test/issues/0468.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  var expected = '- a: a_value\n  b: b_value\n- a: a2_value\n  expanded: true\n  items:\n  - a: a_a_value\n    b: a_b_value\n';
```
Comment (puzrin) [inline test/issues/0468.js]: after `value` => `val`, if line length still too big, add `/* eslint-disable max-len */` above line with problem.

Comment (diberry) [reply test/issues/0468.js]: I will fix.

File: test/issues/0468.js (hunk 1 above)
Comment (diberry) [reply test/issues/0468.js]: I will fix.

**Comment by diberry** [comment]
@puzrin I ran from Makefile and saw the linter errors but wanted feedback before applying linting fixes.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (puzrin) [inline .vscode/launch.json]: I guess, this file is not needed for this PR.

File: test/issues/0468.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+
+var assert = require('assert');
+var yaml = require('../..');
+
+test('should not indent arrays an extra level when disabled', function () {
+  /* eslint-disable max-len */
+  var output = yaml.dump([ { a:'a_val', b:'b_val' }, { a:'a2_val', expanded:true, items: [ { a:'a_a_val', b:'a_b_val' } ] } ], { noArrayIndent: true });
```
Comment (puzrin) [inline test/issues/0468.js]: Please, format this to multiline form first, as been requested (i provided sample). Lint rules can be disabled only when following those is not acceptable. Not this case.

**Comment by diberry** [comment]
#sign-off

File: test/issues/0468.js [hunk 5]
```diff
@@ -6,7 +6,15 @@ var yaml = require('../..');
...
-  var output = yaml.dump([ { a:'a_val', b:'b_val' }, { a:'a2_val', expanded:true, items: [ { a:'a_a_val', b:'a_b_val' } ] } ], { noArrayIndent: true });
+  var output = yaml.dump(
+    [
+      { a:'a_val', b:'b_val' },
+      { a:'a2_val', expanded:true, items:
+        [
+          { a:'a_a_val', b:'a_b_val' }
+        ]
+      }
+    ], { noArrayIndent: true });
```
Comment (puzrin) [inline test/issues/0468.js]:
```js
var output = yaml.dump(
  [
    {
      a: 'a_val',
      b:'b_val'
    },
    {
      a: 'a2_val',
      expanded:true,
      items: [
        {
          a: 'a_a_val',
          b: 'a_b_val'
        }
      ]
    }
  ],
  { noArrayIndent: true }
);
```

1. Change formatting to above please.
2. Is this sample really minimal required for this test? For example, i don't understand what field `expanded` does here.

Comment (diberry) [reply test/issues/0468.js]: fixed

**Comment by diberry** [comment]
@puzrin Please let me know if there are other changes you need.

**Comment by jacob-hd** [comment]
@diberry @puzrin Sorry I missed this edge case... thanks for implementing the fix.
